akd/agents/risk/risk.py

```diff
@@ -366,34 +402,54 @@ def build_dag_from_criteria(
+
+        for rid in irrelevant_risks:
+            auto_pass_node = TaskNode(
+                output_label=f"{rid}_auto_pass",
+                instructions=(
+                    f"This risk ('{rid}') had no applicable criteria for this evaluation. "
+                    f"It must automatically be treated as passed for weighting purposes.\n"
+                    f"Answer strictly with `True`."
+                ),
+                evaluation_params=[],
+                children=[],  # will link later to summary node
+                label=f"{rid} auto-pass node (irrelevant risk)",
+            )
+            final_risk_nodes.append(auto_pass_node)
+            root_nodes.append(auto_pass_node)  # make it part of the DAG roots
+            logger.debug(f"Added auto-pass node for irrelevant risk: {rid}")
 
-        # Creating a weight table for the instruction
-        weight_lines = [f"- {rid}: {weights[rid]}" for rid in criteria_by_risk.keys()]
+        weight_lines = [f"- {rid}: {weights[rid]}" for rid in weights.keys()]
         weights_text = "\n".join(weight_lines)
         denom = sum(weights.values())
+
         if denom == 0:
-            logger.error("Risk weights sum to zero; cannot compute weighted ration.")
-            raise ValueError(
-                "Risk weights sum to zero; cannot compute weighted ration.",
-            )
+            raise ValueError("Risk weights sum to zero; cannot compute weighted ratio.")
 
-        # The child outputs we consult:
+        # Include both evaluated and auto-pass nodes
         all_risk_outputs = [n.output_label for n in final_risk_nodes]
 
         # Bucketing the weighted ratio to 5 verdicts (works for any weights).
         verdicts = [
-            VerdictNode(verdict="Weighted pass ratio ≥ 0.90", score=10.0),
-            VerdictNode(verdict="Weighted pass ratio ≥ 0.75", score=7.5),
-            VerdictNode(verdict="Weighted pass ratio ≥ 0.50", score=5.0),
-            VerdictNode(verdict="Weighted pass ratio ≥ 0.25", score=2.5),
-            VerdictNode(verdict="Weighted pass ratio < 0.25", score=0.0),
+            VerdictNode(verdict="Weighted pass ratio ≥ 0.95", score=10.0),
+            VerdictNode(verdict="Weighted pass ratio ≥ 0.85", score=9.0),
+            VerdictNode(verdict="Weighted pass ratio ≥ 0.75", score=8.0),
+            VerdictNode(verdict="Weighted pass ratio ≥ 0.65", score=7.0),
+            VerdictNode(verdict="Weighted pass ratio ≥ 0.55", score=6.0),
+            VerdictNode(verdict="Weighted pass ratio ≥ 0.45", score=5.0),
+            VerdictNode(verdict="Weighted pass ratio ≥ 0.35", score=4.0),
+            VerdictNode(verdict="Weighted pass ratio ≥ 0.25", score=3.0),
+            VerdictNode(verdict="Weighted pass ratio ≥ 0.15", score=2.0),
+            VerdictNode(verdict="Weighted pass ratio ≥ 0.05", score=1.0),
+            VerdictNode(verdict="Weighted pass ratio < 0.05", score=0.0),
         ]
```

Comment:
can this be maybe configured in a way can be generated automaitcally? like bucketting:
something like

```
RiskAgent._min_risk_score
RiskAgent._max_risk_score
RiskAgent._num_risk_bins
```
We could have this in the riskagentconfig itself....and binning will happen automatically.

I want to remove any hard-coded things that can be set at runtime.